src/test/java/co/cask/cdc/plugins/performance/CDCPipelinePT.java

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdc.plugins.performance;
+
+import co.cask.cdap.etl.api.batch.SparkSink;
+import co.cask.cdap.etl.api.streaming.StreamingSource;
+import co.cask.cdap.etl.proto.v2.ETLPlugin;
+import co.cask.cdap.proto.ProgramRunStatus;
+import co.cask.cdap.test.SparkManager;
+import co.cask.cdc.plugins.common.BigtableOperations;
+import co.cask.cdc.plugins.sink.CDCBigTableConfig;
+import co.cask.cdc.plugins.source.sqlserver.CTSQLServerConfig;
+import co.cask.hydrator.common.Constants;
+import com.google.bigtable.repackaged.com.google.cloud.ServiceOptions;
+import com.google.bigtable.repackaged.com.google.cloud.bigtable.grpc.io.IOExceptionWithStatus;
+import com.google.bigtable.repackaged.io.grpc.StatusRuntimeException;
+import com.google.common.collect.ImmutableMap;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Connection;
+import org.apache.hadoop.hbase.client.Result;
+import org.apache.hadoop.hbase.client.ResultScanner;
+import org.apache.hadoop.hbase.client.Scan;
+import org.apache.hadoop.hbase.client.Table;
+import org.awaitility.Awaitility;
+import org.awaitility.Duration;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class CDCPipelinePT extends CDCPluginPerfTestBase {
```

Comment:
Does PT stand for perf test? I'd prefer if we use the full name, CDCPipelinePerfTest

Reply:
in this case I should also rename all unit tests to UnitTest becase of class name pattern matching for maven task

Reply:
Prefer that if it's not too much work

Reply:
renamed